lua/ui/game/economy.lua

```diff
@@ -482,3 +478,6 @@ end
 function InitialAnimation()
     import(UIUtil.GetLayoutFilename('economy')).InitAnimation()
 end
+
+local LayoutHelpers = import("/lua/maui/layouthelpers.lua")
+local Button = import("/lua/maui/button.lua").Button
```

Comment:
```suggestion

--- Kept for mod backwards compatibility
local LayoutHelpers = import("/lua/maui/layouthelpers.lua")
local Button = import("/lua/maui/button.lua").Button
local options = Prefs.GetFromCurrentProfile('options')

```

Maybe mod backwards compat imports can be lazy imports?
No such thing as lazy importing a class (`Button`) though (which happens often), unless you start doing things like modifying the global/module's metatable.

Reply:
iirc @Garanas wanted to remove lazy imports completely.